.stickler.yml

```diff
@@ -0,0 +1,10 @@
+---
+linters:
+  phpcs:
+      standard: 'dev/setup/codesniffer/ruleset.xml'
```

Comment:
I am currently validating that this is the right way to define a custom ruleset. I have [twitted](https://twitter.com/jtraulle/status/1166072293497483264) to @sticklerci and I am currently waiting for a reply.

Maybe we need to add a symbolic link `phpcs.xml` at the root of the project that direct to `dev/setup/codesniffer/ruleset.xml`. [Doc](https://stickler-ci.com/docs#php) is unclear to me, that's why I asked for confirmation [on Twitter](https://twitter.com/jtraulle/status/1166072293497483264).

Reply:
[It is OK](https://twitter.com/sticklerci/status/1166138847035613184) :slightly_smiling_face: